_includes/pbc/all/install-features/202505.0/ssp/install-ssp-service-management-features.md

```diff
@@ -0,0 +1,1173 @@
+| SspServiceMenuItemWidget                | Adds a menu item for accessing SSP services in the navigation.               |            | SprykerFeature\Yves\SspServiceManagement\Widget |
+| SspServiceChangeScheduledTimeLinkWidget | Provides a link to change the scheduled time for a specific service.         |            | SprykerFeature\Yves\SspServiceManagement\Widget |
+| ShipmentTypeServicePointSelectorWidget  | Displays a selector for choosing a service point based on the shipment type. |            | SprykerFeature\Yves\SspServiceManagement\Widget |
+| ServicePointNameForItemWidget           | Displays the name of the service point associated with a specific cart item. |            | SprykerFeature\Yves\SspServiceManagement\Widget |
+| ListItemsByShipmentTypeWidget           | Lists items in the cart grouped by their shipment type.                      |            | SprykerFeature\Yves\SspServiceManagement\Widget |
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+use SprykerFeature\Yves\SspServiceManagement\Widget\ListItemsByShipmentTypeWidget;
+use SprykerFeature\Yves\SspServiceManagement\Widget\ServicePointNameForItemWidget;
+use SprykerFeature\Yves\SspServiceManagement\Widget\ShipmentTypeServicePointSelectorWidget;
+use SprykerFeature\Yves\SspServiceManagement\Widget\SspServiceChangeScheduledTimeLinkWidget;
+use SprykerFeature\Yves\SspServiceManagement\Widget\SspServiceMenuItemWidget;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            SspServiceMenuItemWidget::class,
+            SspServiceChangeScheduledTimeLinkWidget::class,
+            ShipmentTypeServicePointSelectorWidget::class,
+            ServicePointNameForItemWidget::class,
+            ListItemsByShipmentTypeWidget::class,
+        ];
+    }
+}
+```
+
+2. Enable Javascript and CSS changes:
+
+```bash
+console frontend:yves:build
+console frontend:zed:install-dependencies
+console frontend:zed:build
+```
+
+{% info_block warningBox "Verification" %}
+
+1. On the Storefront, use the search to find the service product you've created.
+  Make sure the product is available in the catalog.
+
+2. Click the product to open its details page. Make sure the following applies on the product details page:
+  - A service point selector is displayed
+  - A date and time selector is displayed
+  - Delivery and service-on-site shipment types are available
+
+3. Select a service point. 
+4. Select a service date and time.
+5. Add the product to cart.  
+6. Go to the cart page and make sure the following applies:
+  - The cart item displays the selected service point
+  - Items are grouped by shipment type
```

Comment:
Don't we have to add more than one item to properly verify this?

Reply:
Yes, add the sentence on top.